tests/test_chat_api.py

```
@@ -0,0 +1,483 @@
+def test_create_session_nonexistent_author(test_client, auth_headers):
+    """Test creating session for non-existent author."""
+    response = test_client.post(
+        "/api/v1/chat/nonexistent-author/sessions",
+        headers=auth_headers,
+    )
+
+    assert response.status_code == 404
+    assert "not found" in response.json()["detail"].lower()
+
+
+def test_list_chat_sessions_empty(test_client, auth_headers, test_author_id):
```

Ruff (0.14.3)

Unused function argument: `test_author_id`

(ARG001)